native/cocos/editor-support/spine-wasm/spine-type-export.cpp

```diff
@@ -316,7 +316,7 @@ EMSCRIPTEN_BINDINGS(spine) {
     class_<Event>("Event")
         .constructor<float, const EventData &>()
         .function("getData", optional_override([](Event &obj) {
-            return &obj.getData(); }), allow_raw_pointers())
+            return obj.getData(); }), allow_raw_pointers())
```

Comment:
Why do we need to change the return value from `const EventData *` to `const EventData&`?
Did it cause errors? If so, what kind of errors?